src/WebJobs.Extensions.DurableTask/DefaultPlatformInformation.cs

```diff
@@ -40,7 +35,7 @@ public bool InLinuxConsumption()
             return inLinuxConsumption;
         }
 
-        public bool InLinuxAppService()
+        private bool InLinuxAppService()
         {
             string azureWebsiteInstanceId = this.nameResolver.Resolve("WEBSITE_INSTANCE_ID");
```

Comment:
You could pull this out in another method to make it clear what azureWebsiteInstanceId and inAppService are used for. I see you use them in both of these Linux methods.

And if this variable determines if it's consumption or not, why do you need to call InLinuxConsumption and InWindowsConsumption in the method IsInConsumption? It seems like you're using this variable to determine that already.